Task_HelloWorld/index.html

```diff
@@ -1,4 +1,3 @@
-<!DOCTYPE html>
 <html>
   <head>
```

Comment:
The text 'coder its' is not wrapped in any tag. It should be wrapped in a valid HTML tag.